daprdocs/content/en/developing-applications/ides/visual-studio-code/vscode-manual-configuration.md

```diff
@@ -224,4 +171,4 @@ Once you have made the required changes, you should be able to switch to the [de
 
 {{% alert title="Note" color="primary" %}}
 Since you didn't launch the service(s) using the **dapr** ***run*** cli command, but instead by running **daprd**, the **dapr** ***list*** command will not show a list of apps that are currently running.
```

Comment:
Read through this document and make sure this is still relevant and works. For example using the CLI to generate a components folder is wrong here (this is an old feature). It looks like this section needs updating. At the same time remove all usage of the word "will" it is better to write in the present tense, rather than the future tense.

Reply:
Sounds good, I should've checked that.

Reply:
Finished checking that the information is accurate, it's pretty much what the scaffold task in the extension does. It doesn't rely on Dapr instead uses daprd to to launch a service when the debug play button is used. Removed the 'will' from every spot except were it makes sense in the disclaimer at the bottom talking about the Dapr List lack of identifying daprd instances. Resolving this.